services/director-v2/tests/conftest.py

```diff
@@ -38,11 +35,9 @@
 from simcore_service_director_v2.models.domains.comp_tasks import CompTaskAtDB, Image
 from simcore_service_director_v2.utils.computations import to_node_class
 from sqlalchemy import literal_column
+from sqlalchemy.sql.expression import select
 from starlette.testclient import TestClient
 
-nest_asyncio.apply()
```

Comment:
Nice, you managed to remove this

Reply:
actually did not understand its purpose...